apps/dashboard/mixins.py

```diff
@@ -123,6 +123,27 @@ def post(self, request, *args, **kwargs):
                         organisation_slug=self.organisation.slug)
 
 
+class DashboardUserRemovalMixin:
```

Comment:
Can you not merge this one and the `ModRemovalMixin` into one? Maybe determine the action via the `submit_action` value, set in the submit button.